src/runtime/fake_thread_pool.cpp

```diff
@@ -78,6 +78,21 @@ WEAK void halide_mutex_lock(halide_mutex *mutex) {
 WEAK void halide_mutex_unlock(halide_mutex *mutex) {
 }
 
+WEAK halide_mutex_array* halide_mutex_array_create(int sz) {
```

Comment:
Didn't notice this was the fake_thread_pool and wondered for a long time how this could possibly work :)

Reply:
Added a comment to avoid confusion from future readers.